### DISCUSSION TITLE
How to change column head mouse-over content

### DISCUSSION THREAD

**Discussion by mcadotsch** [KevinVandy/material-react-table#128] [Q&A] [answered]
How can i change the default content of the mouse-over textbox (tooltip) in the column header?

**Comment by KevinVandy** [comment, marked as answer]
The title is just set to the `header` value on a column definition for accessibility reasons when the header gets cut off. You could add your own `<Tooltip>` around a header in the `Header` custom render if you want.

https://www.material-react-table.com/docs/guides/data-columns#custom-header-render

**Reply by mcadotsch** [reply]
Thnx! That did the trick.

**Comment by jjwallaby** [comment]
Hi,
I tried this... But seems there is still a default ToolTip occurring which displays a second tooltip with the name of the column field.

columns modified :- 

      {
        accessorKey: 'email',
        header: 'Email',
        title: 'this is the main title',
        Header: ({ column }) => (
          <Tooltip title={"this is override title of the Column Field. However  it seems that there is a second tooltip enable"}>
          <i style={{ color: 'red' }}>{column.columnDef.header}</i>

</Tooltip>
        ),
        muiEditTextFieldProps: {
          type: 'email',

          required: true,
          error: !!validationErrors?.email,
          helperText: validationErrors?.email,
          //remove any previous validation errors when user focuses on the input
          onFocus: () =>
            setValidationErrors({
              ...validationErrors,
              email: undefined,
            }),
        },

![Screenshot from 2024-01-23 14-11-04](https://github.com/KevinVandy/material-react-table/assets/11285975/3cf5b421-fb0f-4a44-ad03-b232bdd74be2)

"material-react-table": "^2.8.0",
